weave-js/src/components/Panel2/WeaveFormatContext.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type PanelNumberFormat = {
+  textAlign?: React.CSSProperties['textAlign'];
+  padding?: React.CSSProperties['padding'];
+  justifyContent?: string;
```

Comment:
Could the types on justify and align also be tightened with `React.CSSProperties`?

Reply:
done